.drone.yml

```diff
@@ -84,13 +84,19 @@ pipeline:
     commands:
       - /opt/docker/bin/rnode -c ./drone/config/bootstrap.toml run
 
+  wait-for-bootstrap:
+    image: docker
+    commands:
+      - until ci/is-container-log-line-present.sh _step_4 'Making a transition to ApprovedBlockRecievedHandler state.'; do echo $?; sleep 5; done
```

Comment:
Does drone yaml version support constants? `Making a transition to ApprovedBlockRecievedHandler state.` is a good candidate

Reply:
I don't believe it does